content/project/board-election-process.adoc

```diff
@@ -25,178 +25,165 @@ A board comprised fully of write-in candidates runs the risk of being overweighe
 . Make sure that the board reflects the balance of different viewpoints — the needs of users; of committers; of organizations, both large and small; of commercial interests.
 . Maintain stability and avoid sudden direction changes.
 
-== 2020 Elections
+== 2021 Elections
 
-NOTE: Jenkins 2020 Elections are over, thanks to all participants!
-Please see the link:/blog/2020/12/03/election-results[results announcement].
+NOTE: Jenkins 2021 Elections are over, thanks to all participants!
+Please see the link:/blog/2021/12/03/election-results[results announcement].
 
-During the 2020 elections we elected two governing board members and five officers, namely:
+During the 2021 elections, we elected two governing board members and five officers, namely:
 link:/project/team-leads/#security[Security], link:/project/team-leads/#events[Events], link:/project/team-leads/#release[Release], link:/project/team-leads/#infrastructure[Infrastructure], and link:/project/team-leads/#documentation[Documentation].
 The terms of office for these elected positions are:
 
-* Officer positions (1 year): December 03, 2020 to December 2, 2021
-* Governing board member (2 years): December 03, 2020 to December 2, 2022
+* Officer positions (one year): December 03, 2021 to December 2, 2022
+* Governing board member (two years): December 03, 2021 to December 2, 2023
 
 === References
 
-* link:/blog/2020/12/03/election-results[Election results]
-* link:/blog/2020/10/28/election-candidates[Election candidates]
-* link:/blog/2020/09/24/board-elections/[Election announcement]
+* link:/blog/2021/12/03/election-results[Election results]
+* link:/blog/2021/10/25/jenkins-elections/[Voter registration]
+* link:/blog/2021/09/20/election-period-opened/[Election announcement]
 
 === Key dates
 
-* **Sep 24** - Nominations open
-* **Sep 24** - Voting sign-up begins
-* **Oct 15** - Nominations deadline
+* *October 20* - Nominations open
+* *October 20* - Voting registration begins
+** During the registration period, the election committee will monitor the sign-ups and update the list of voters.
+* *November 10* - Nominations deadline
 ** After the deadline, the election committee will process nominations and notify the candidates
-* **Oct 26** or later - Election candidates are published, including personal statements
-* **Nov 10** - Voting begins
+* *November 17* - Voting registration is closed
+* *November 17* - Election candidates are published, including personal statements
+* *November 17* - Voting begins
 ** Email is distributed to all registered voters through link:https://civs.cs.cornell.edu/[Condorcet Internet Voting Service]
-* **Nov 08** - Voting sign-up is over
-** During the registration period, the election committee will monitor the sign-ups and update the list of voters.
-* **Nov 27** - Voting ends, 11PM UTC
-* **Dec 03** - Results are announced and take effect.
+* *December 02* - Voting ends, 11PM UTC
+* *December 07* - Results are announced.
```

Comment:
Small thing, but maybe remove the period here for consistency with the other date items.

Reply:
ah good call, removing!